src/trackers/classes/site.js

```diff
@@ -31,6 +35,22 @@ class Site {
         this.cnameCloaks = {}
 
         this.analyzeRequest = _analyzeRequest.bind(this)
+
+        this.documentCookies = Object.values(siteData.data.apis.savedCalls
```

Comment:
I think we need to name this better / drop a comment. From the name I thought those are all cookies (like in `document.cookies`), but those are only 1p cookies set by 3p scripts, correct? Maybe `thirdPartyJSCookies`?